tests/acceptance/features/bootstrap/WebUIGuestsContext.php

```diff
@@ -55,21 +56,62 @@ class WebUIGuestsContext extends RawMinkContext implements Context {
 	 * @var EmailContext
 	 */
 	private $emailContext;
+
 	/**
 	 *
 	 * @var SetPasswordPage
 	 */
 	private $setPasswordPage;
 
+	/**
+	 *
+	 * @var PreUserAddDialog
+	 */
+	private $preUserAddDialog = 'Add ';
+
+	/**
+	 *
+	 * @var PostUserAddDialog
+	 */
+	private $postUserAddDialog = ' (guest)';
+
+	/**
+	 *
+	 * @var string
+	 */
+	private $userAddDialogBoxFramework = "%s%s%s";
+
+	/**
+	 *
+	 * @var $FilesPage
```

Comment:
```suggestion
	 * @var FilesPage $filesPage
```

Reply:
I followed the style that's already in legacy code:
	 * @var FeatureContext
	 */
	private $featureContext;

	/**
	 *
	 * @var WebUIGeneralContext
	 */
	private $webUIGeneralContext;

	/**
	 *
	 * @var GuestsContext
	 */
	private $guestsContext;

Reply:
Okay, that works too. But, there's additional `$` there. You can also write following:

```suggestion
	 * @var FilesPage
```